browser/js/global.js

```diff
@@ -1595,11 +1595,16 @@ function getInitializerClass() {
 					global.docURL.includes('data/idle/calc/') ||
 					global.docURL.includes('data/multiuser/calc/');
 
+				// FIXME: it doesn't work in calc under cypress
```

Comment:
slight nit: may you clarify what "it" is here ... I presume you mean "accessibility mode" but I'm not entirely sure